documentation/how-to/prepare-for-production.md

```diff
@@ -0,0 +1,193 @@
+
+# Prepare for Production
+
+## Performance Optimisation
```

Comment:
Seeing the content, now it is clear to me we have to split this into two how-to's. One for performance optimization and another one for security. For the titles, try to think what the user would out of them, what problem they would solve using these guides. One can be "how to optimise your deployment performance" or so, but for the other I am not sure yet. It cannot be as simple as "Security".

Reply:
There are 2 topics which don't fit under either: database backups and observability. I don't know that these can be grouped. Should this be 4 how-tos? The backups topic is short as this is already covered in other documentation and I didn't want to repeat it and risk the documentation I write becoming outdated when they make an update to theirs. But I could do add an example here if it would be helpful